src/index.html

```diff
@@ -6,7 +6,41 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
-  </body>
+  <body class="page">
+    <main class="main page__main-block">
+      <div class="calendar main__calendar">
+        <div class="day calendar__day day--month-length--31 day--start-day--thu"></div>
+        <div class="day calendar__day day--month-length--31"></div>
```

Comment:
leave just one class on the children

```suggestion
        <div class="day calendar__day"></div>
```